src/MCPServer/debian/postinst

```diff
@@ -13,9 +21,14 @@ else
   rsync -a /usr/share/archivematica/sharedDirectoryStructure/* /var/archivematica/sharedDirectory/.
 fi
 
-chown -R archivematica:archivematica /var/archivematica/
+chown -R archivematica:archivematica /var/archivematica/sharedDirectory
```

Comment:
Add the trailing `/` - if `/var/archivematica/sharedDirectory` is a symlink this will only change permissions on the symlink not the directory.